### PR TITLE
Fix typos in Issue Template

### PR DESCRIPTION
**Pull request by BasThomas**
## Proposed changes

Fix some typos in the Issue Template and make some sentences easier to read.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Create a [gist](https://gist.github.com) which is a paste of your _full_ Appium
 
 ## Code To Reproduce Issue [ Good To Have ]
 
-Please remember that, with sample code; it's easier to reproduce bug and much faster to fix it.
+Please remember that, with sample code; it's easier to reproduce the bug and it's much faster to fix it.
```
Comment (imurchie) [inline .github/ISSUE_TEMPLATE.md]: Since we're here, can you get rid of the comma before and the semicolon after "with sample code"?

**Review by imurchie — CHANGES_REQUESTED** [review]
Thanks for this!

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by imurchie** [comment]
Thanks! Looks good! Once you've signed the CLA we can get this in!

**Review by imurchie — APPROVED** [review]

**Comment by BasThomas** [comment]
I did sign it; does it check your GitHub email?

**Comment by imurchie** [comment]
Yes. It is based on your email in GitHub.

**Comment by BasThomas** [comment]
🤔. So the CLA signing seems to be working through clabot's url (https://cla.js.foundation/appium/appium?pullRequest=7822), but not via the url in the PR template (the Google docs one)

**Comment by imurchie** [comment]
Hmm. Yes, that ought to be changed. :(

**Comment by imurchie** [comment]
https://github.com/appium/appium/pull/7823

**Comment by imurchie** [comment]
Thanks for this!

**Pull request merged**